app/facades/movie_facade.rb

```diff
@@ -1,29 +1,29 @@
 class MovieFacade
   def self.search(title)
     movies = []
-    page = 1
     until movies.count >= 40
-      movies_data = MovieService.find_by_title(title, page)
+      movies_data = MovieService.search_by_title(title, page ||= 1)
       movies_data.each do |search_data|
         movies << MoviePoro.new(search_data)
       end
-      # leave comment here explaining the usage of the number 20
+      # the magic number 20 is used here, because that is the default yield per page from this API
```

Comment:
Could we have stored this magic number in a variable which could have explained its purpose?

Reply:
That's a great observation. I think you're 100% right here. This could be a good opportunity for something to bring up in the eval for the 'Team can identify areas where code can be refactored and how they may have created technical debt' requirement in the rubric.